lib/ifc-utils.ts

```
@@ -1614,24 +1615,61 @@ export function exportData(
+          const mesh = new Mesh(geo, new MeshStandardMaterial());
+          scene.add(mesh);
+        }
+      });
+      const exporter = new GLTFExporter();
+      const res = await exporter.parseAsync(scene, { binary: true });
+      if (res instanceof ArrayBuffer) {
+        return res;
+      }
+      return new TextEncoder().encode(JSON.stringify(res)).buffer as ArrayBuffer;
+    }
+    case "csv":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)